lib/zold/node/front.rb

```diff
@@ -314,5 +314,9 @@ def score
       raise 'Score is empty, there is something wrong with the Farm!' if best.empty?
       best[0]
     end
+
+    def http_score_header
```

Comment:
@theterminalguy this a bad idea. You are not supposed to test internals of an object.